_posts/2018-02-02-rebble-web-services.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Pebbling after Fitbit: introducing the Rebble Web Services
+date:   2018-02-02 11:42:42
+# categories: community
+---
+
+Hey there! [Katharine Berry](https://twitter.com/KatharineBerry) here. As you may well have heard
+by now, Fitbit will be shutting down all Pebble services at the end of June of this year. At that
+point, no services that require a Pebble server will work any more — which includes the appstore,
+weather, dictation, some parts of timeline, iOS SMS/email actions, the CloudPebble IDE, and more.
+
+The observant among you might also notice that none of these things have been mentioned in a
+community update in over a year. Fortunately, we have a plan: the Rebble Web Services.
+
+_(Disclaimer: absolutely everything in this post is subject to change)_
+
+------
+
+# Funding
+
+Running these services will cost money. Potentially, quite a lot of money. To that end, we will be
+setting up a Patreon page, so that the generous among you can donate to keep the services running.
+Some services are particularly expensive to provide, and so for those we will be requiring that
+a small sum of money be pledged each month. This will enable us to both provide you with the service
+you want, and to support it sustainably for everyone. If you are generous and inclined to donate
+more, we welcome that as well.
+
+# What to expect
+
+## Apps and firmware
+
+You will continue to use Pebble's latest Pebble mobile apps and Pebble firmware for the forseeable
+future. All of the Rebble Web Services will work with these; you'll just have to create a Rebble
+account and tap a button to switch.
+
+## Accounts
+
+Once Fitbit shuts down the Pebble services, you won't even be able to log in! We are going to
+replace your Pebble logins with brand new Rebble logins. Because nobody wants to have _yet more_
+accounts on the internet, we will back your Rebble login with a Google, Facebook, or similar login
+(we have yet to finalise a list).
+
+**For KiezelPay users**: once you switch to using a Rebble account, you will have to re-claim your
+apps using the email address you originally used to purchase the app. This should only be necessary
+once, after which your apps should continue to work as they did before. You will _not_ have to buy
+them again.
+
+## Appstore
+
+As mentioned in [previous community updates](http://rebble.io/2017/01/31/rebble-community-update-2.html),
+work started on a Pebble Appstore replacement almost as soon as Pebble announced that it was ceasing
+operations. The Rebble Appstore should be ready to go by the time the original Pebble appstore
```

Comment:
I know it makes almost no difference but the store is called the `Rebble Store`, not Rebble Appstore. We have to be consistent in our product naming (as you may have noticed some of my other hobbies are video production and design, and that has led me to get a bit into marketing and social media management and learn that small things like this actually make a difference.)

Reply:
Why is it the "Rebble Store", though? If we're going to be consistent, we may as well be consistent with what we're replacing, too.

Reply:
You're right about that but originally it was called `The Panic Store`, but we had to change that name due to Panic the Mac/iOS software developer and came up with a simple name `Rebble Store`, I left out App because that's mostly an Apple thing `App Store` (Google calls it Play Store). So as I said before, mostly a marketing thing and consistency with the name in the navbar.